composer.json

```diff
@@ -8,7 +8,7 @@
     "prefer-stable": true,
     "require": {
         "php": "~5.5|~7.0",
-        "doctrine/dbal": "~2.5.5",
+        "doctrine/dbal": "~2.6.3",
```

Comment:
any reason not to required just any 2.6 by `~2.6` ?

Reply:
@seut ~2.6.3 will allow all hotfix versions > 2.6.3 but < 2.7 so I'm not sure if I got your questions right.

Reply:
ah right, ignore ;)